up.py

```diff
@@ -7,15 +7,18 @@
 
 parser = argparse.ArgumentParser()
 group = parser.add_mutually_exclusive_group()
-group.add_argument("--setup", help="initial setup", action='store_true')
-group.add_argument("--start", help="start the app", action='store_true')
-group.add_argument("--stop", help="stop the app", action='store_true')
-group.add_argument("--restart", help="Restart the container without rebuilding", action='store_true')
+group.add_argument("--setup", help="initial setup", action="store_true")
+group.add_argument("--start", help="start the app", action="store_true")
+group.add_argument("--stop", help="stop the app", action="store_true")
+group.add_argument(
```

Comment:
its obviously not perfect, but the extra whitespace added here that breaks with the rest of the code above, just looks odd.